onnxruntime/python/tools/transformers/models/phi2/README.md

```diff
@@ -0,0 +1,119 @@
+# Phi2 Optimizations
+## Prerequisites
+A Linux machine for [TorchDynamo-based ONNX Exporter](https://pytorch.org/docs/stable/onnx.html#torchdynamo-based-onnx-exporter)\
+Install onnx, onnxscript and transformers by running
+```bash
+pip install -r requirements.txt
+```
+To export ONNX, PyTorch version 2.2.0 or higher is required. The [official website](https://pytorch.org/) offers packages compatible with CUDA 11.8 and 12.1. Please select the appropriate version according to your needs.
+\
+\
+**There are two options to run the conversion script:**\
+_From source:_
+```bash
+pip install onnxruntime-gpu==1.17.0 # or onnxruntime==1.17.0 if using cpu

```

Comment:
May add a link https://onnxruntime.ai/docs/install/#python-installs for onnxruntime installation (That covers CUDA 12 package as well).

Reply:
sure, will add in seperate PR